actions/rules/action_field_set_test.go

```diff
@@ -1,135 +1,191 @@
 package rules
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 
 	"github.com/fabric8-services/fabric8-wit/actions/change"
 	"github.com/fabric8-services/fabric8-wit/gormtestsupport"
-	"github.com/fabric8-services/fabric8-wit/resource"
 	tf "github.com/fabric8-services/fabric8-wit/test/testfixture"
 	"github.com/fabric8-services/fabric8-wit/workitem"
+	"github.com/jinzhu/copier"
```

Comment:
Okay, that is not good. Maybe we can just create two work items using the test fixture and overwrite the ID to of the second with the one from the first. That is effectively the same as doing a deep copy, well except for things like the number being different, but this can be overwritten as well. If we absolutely need to copy a work item we should maybe implement and test this method on the work item code itself. I just want to avoid that we forget fields.